### DISCUSSION TITLE
Neuen Darstellungsfilter anlegen

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#74] [Q&A]
Beim Anlegen eines neuen Darstellungsfilters in den Builds 5.23.1901 und 5.23.2001 bekomme ich nach Klicken von "Übernehmen" folgende Fehlermeldung:

`NullReferenceException:
Object reference not set to an instance of an object.`

Workaround ist derzeit, einen bestehende Darstellungsfilter kopieren und bearbeiten - das geht.

**Comment by gis-eni** [comment]
Bug. Ist mit Build 5.23.2101 behoben
